run/README.md

```diff
@@ -101,9 +101,22 @@ For more Cloud Run samples beyond Node.js, see the main list in the [Cloud Run S
 
     4. Exit the container: `Ctrl-D`
 
-## Deploying
+## Running the Tests
+
+```sh
+# Run unit & integration tests.
+npm test
 
+# Run system tests.
+SAMPLE=[SAMPLE_TO_TEST]
+gcloud builds submit --tag gcr.io/$GOOGLE_CLOUD_PROJECT/${SAMPLE}:manual
```

Comment:
Do we set GOOGLE_CLOUD_PROJECT anywhere in the README or do we assume they have it set?

Reply:
Probably assumed. But as the README is not a tutorial, I felt it was reasonable to rely on gcloud to throw an error. I can add something like the SAMPLE var just above if you think that would help.